Resources/doc/view_response_listener.rst

```diff
@@ -72,6 +72,25 @@ explicit configuration. So make sure to remove or disable the following setting:
         }
     }
 
+If you need to pass additional data to template, not for serialization,
+you should return a $view object with the data set by ``setTemplateData``.
+
+.. code-block:: php
+
+    <?php
+
+    /**
+     * @View()
+     */
+    public function getUsersAction()
+    {
+        // ...
+        $view = $this->view($data, 200)
```

Comment:
hmm actually it might be clearer to use ``new View`` here as ``$this->view`` isnt available by default